src/bus/mod.rs

```diff
@@ -0,0 +1,15 @@
+use core::fmt::Debug;
+
+mod four_wire;
+mod three_wire;
+
+pub use self::four_wire::FourWire;
+pub use self::three_wire::ThreeWire;
```

Comment:
Not sure what you mean, `bus` is public.

Reply:
[This](https://github.com/kellerkindt/w5500/blob/master/src/bus/mod.rs#L8) wasn't public before. Otherwise one could not work with the FourWireError type?